packit_service/worker/reporting.py

```diff
@@ -95,9 +96,16 @@ def set_status(
             url = "https://wiki.centos.org/Manuals/ReleaseNotes/CentOSStream"
 
         logger.debug(f"Setting status for check '{check_name}': {description}")
-        self.project.set_commit_status(
-            self.commit_sha, state, url, description, check_name, trim=True
-        )
+
+        try:
+            self.project.set_commit_status(
+                self.commit_sha, state, url, description, check_name, trim=True
+            )
+        except gitlab.exceptions.GitlabHttpError:
+            # Ignoring Gitlab 'enqueue' error
+            # https://github.com/packit-service/packit-service/issues/741
+            pass
```

Comment:
What is done when we get that exception? Nothing is changed on GitLab side?

What can we do instead to show that info to the user? What about having a unique status for each step of our workflow? e.g. `packit-service/copr-build-srpm`, `packit-service/copr-build-fedora-31-rpm` (We do that in the past..;)

Reply:
Do we want to solve this here or make a separate PR for this. We have multiple possibilities for that so it's worth a separate discussion.

Reply:
I will make a different pr for this.

Reply:
Thanks!